internal/provider/webhook_recipient_resource.go

```diff
@@ -34,6 +37,10 @@ var (
 
 	webhookTemplateTypes     = []string{"trigger", "exhaustion_time", "budget_rate"}
 	webhookTemplateNameRegex = regexp.MustCompile(`^[a-z](?:[a-zA-Z0-9]+$)?$`)
+
+	defaultHeaderContentType            = "Content-Type"
+	defaultHeaderUserAgent              = "User-Agent"
+	defaultHeaderXHoneycombWebhookToken = "X-Honeycomb-Webhook-Token"
```

Comment:
if you pull these into a `[]string` (similar to to `webhookTemplateTypes` above) you can make use of [stringvalidator.NoneOfCaseInsensitive](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-validators@v0.15.0/stringvalidator#NoneOfCaseInsensitive) and remove 276-286 in `ValidateConfig`

Reply:
oooh nice! I wrongly assumed there wouldn't be a case insensitive built in